src/main/java/offeneBibel/osisExporter/ObOsisGeneratorVisitor.java

```diff
@@ -173,9 +180,26 @@ else if(node.getNodeType() == ObAstNode.NodeType.verse) {
                 m_currentFassungContainsVerses = true;
                 m_skipVerse = false;
                 m_verseTag = m_verseTagStart + verse.getNumber();
-                m_currentFassung += "<verse osisID=\"" + m_verseTag + "\" sID=\"" + m_verseTag + "\"/>";
+                m_currentFassung.append("<verse osisID=\"" + m_verseTag + "\" sID=\"" + m_verseTag + "\"/>");
+                if (m_inlineVersStatus) {
+                    ObVerseStatus status = verse.getStatus();
+                    ObVerseStatus leseStatus = verse.getStatus(FassungType.lesefassung);
+                    ObVerseStatus studienStatus = verse.getStatus(FassungType.studienfassung);
+                    String verseStatus;
+                    if (leseStatus == studienStatus || status == studienStatus) {
```

Comment:
So rein interessehalber: Wenn ich das richtig verstehe, packst du in beide Fassungen (sowohl Lese- als auch Studienfassung) die Versstati (Versstatuus? Versstatusse? Versstatus?). Warum nicht in die Lesefassung den Lesefassungsstatus und in die Studienfassung den Studienfassungsstatus?